support-frontend/assets/components/directDebit/directDebitForm/directDebitForm.tsx

```diff
@@ -129,19 +134,19 @@ function DirectDebitForm(props: PropTypes) {
 			<PaymentButton
 				buttonText={props.buttonText}
 				phase={props.phase}
-				onPayClick={props.payDirectDebitClicked}
-				onEditClick={() => props.setDirectDebitFormPhase('entry')}
+				onPayClick={props.confirmAccountDetails}
+				onEditClick={() => props.setPhase('entry')}
 				onConfirmClick={() =>
-					props.confirmDirectDebitClicked(props.onPaymentAuthorisation)
+					props.payWithDirectDebit(props.onPaymentAuthorisation)
```

Comment:
I think `confirmAccountDetails` and `payWithDirectDebit` are much better and clearer names than `payDirectDebitClicked` and `confirmDirectDebitClicked`. I wonder if we should update the names of the `PaymentButton` props too to align with these new names?